services.yaml

```diff
@@ -1645,6 +1645,37 @@ paths:
           $ref: "#/responses/UnauthorizedError"
         403:
           $ref: "#/responses/ForbiddenError"
+  /services/{env_id}/alerts:
+    get:
+      tags:
+        - Alerts
+      summary: "Fetch Active Alerts for an Environment"
+      description: "Use this call to fetch all active alerts on current environment. admins can use the query parameter to fetch alerts which were set by other member/admin"
```

Comment:
not sure i understand from this what happens if i'm calling this as a normal user.

Reply:
you get all alerts that you can see on an environment.